opengrok-indexer/src/main/java/org/opengrok/indexer/history/FileHistoryCache.java

```diff
@@ -138,6 +141,10 @@ private void doFileHistory(String filename, List<HistoryEntry> historyEntries,
         }
 
         storeFile(hist, file, repository, !renamed);
+
+        statRepoHist.report(LOGGER, Level.FINER,
+                String.format("Done storing history cache for '%s'", filename),
```

Comment:
Here, the `String.format()` will be evaluated every time. Thinking of introducing static method `Statistics.eligible(LogLevel level)` that would do something like:
```java
return (LOGGER.isLoggable(level) || Metrics.getRegistry());
```